sss_cal_mgh.dat

```diff
@@ -0,0 +1,306 @@
+113	-0.106600 0.046400 -0.060400 -0.012700 0.005700 -0.999903 -0.186801 -0.982403 -0.003300 -0.981020 0.192489 0.022924 -0.008282
```

Comment:
this is the wrong calibration files. I will send you the link to the correct ones.